pos_partner_deselection/README.rst

```diff
@@ -2,11 +2,11 @@
    :target: https://www.gnu.org/licenses/lgpl
    :alt: License: LGPL-3
 
-==================
- Deselect Partner
-==================
+====================
+Partner  deselection
+====================
```

Comment:
===============
 How It Should Be 
===============

==========
How it's done
==========

I'm about indents.
one `=` before the name and one after

Reply:
got it